diffkemp/diffkemp.py

```diff
@@ -54,6 +57,33 @@ def run_from_cli():
             second_mods = second_builder.build_modules_with_params(
                 args.rebuild)
 
+        if args.default_value:
+            for module_name, module_src in first_mods.iteritems():
+                module_dst = second_mods.get(module_name)
+                if not module_dst:
+                    continue
+
+                module_src.collect_all_parameters()
+                module_dst.collect_all_parameters()
+
+                print module_name
+                if args.param:
+                    param_src = module_src.params.get(args.param)
+                    param_dst = module_dst.params.get(args.param)
+                    if param_src and param_dst:
+                        print "{}: {}".format(args.src_version,
+                                param_src.default_value)
+                        print "{}: {}".format(args.dest_version,
+                                param_dst.default_value)
+                    return 0
+
+                for param_name, param in module_src.params.iteritems():
+                    if module_dst.params.get(param_name):
+                        print "{}: {}".format(args.src_version, param.default_value)
+                        print "{}: {}".format(args.dest_version,
+                                module_dst.params[param_name].default_value)
```

Comment:
This looks very similar to the code 7 lines above. Wouldn't it be better to merge it somehow?
Moreover, it is not necessary to create a whole new if branch for the new command line argument. I would prefer to put this functionality to the existing code (we can get the default values every time when we do the analysis).